# Generated by Django 2.1.5 on 2019-02-02 21:52

import django.contrib.auth.models
import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):
    dependencies = [
        ('rddt_main', '0001_initial'),
    ]

    operations = [
        migrations.AlterModelOptions(
            name='reddituser',
            options={'verbose_name': 'Пользователь', 'verbose_name_plural': 'Пользователи'},
        ),
        migrations.AlterModelManagers(
            name='reddituser',
            managers=[
                ('objects', django.contrib.auth.models.UserManager()),
            ],
        ),
        migrations.AlterField(
            model_name='reddituser',
            name='about',
            field=models.TextField(max_length=400, verbose_name='О себе'),
        ),
        migrations.AlterField(
            model_name='reddituser',
            name='email',
            field=models.EmailField(max_length=100, verbose_name='Электронная почта'),
        ),
        migrations.AlterField(
            model_name='reddituser',
            name='phone_number',
            field=models.CharField(max_length=16, validators=[django.core.validators.RegexValidator('^\\+?\\d{9,15}$',
                                                                                                    'Введите номер телефона в формате +99999999999')],
                                   verbose_name='Номер телефона'),
        ),
        migrations.AlterField(
            model_name='reddituser',
            name='username',
            field=models.CharField(error_messages={'unique': 'Пользователь с таким именем уже существует'},
                                   max_length=30, unique=True, verbose_name='Имя пользователя'),
        ),
    ]
